language/orion/host.go

```diff
@@ -15,16 +15,16 @@ import (
 
 	"github.com/aspect-build/orion/common/bazel/workspace"
 	BazelLog "github.com/aspect-build/orion/common/logger"
-	plugin "github.com/aspect-build/orion/language/host/plugin"
-	starzelle "github.com/aspect-build/orion/language/host/starzelle"
+	plugin "github.com/aspect-build/orion/language/orion/plugin"
+	starzelle "github.com/aspect-build/orion/language/orion/starzelle"
 	"github.com/bazelbuild/bazel-gazelle/config"
 	"github.com/bazelbuild/bazel-gazelle/label"
 	gazelleLanguage "github.com/bazelbuild/bazel-gazelle/language"
 	"github.com/bazelbuild/bazel-gazelle/rule"
 	"github.com/emirpasic/gods/sets/treeset"
 )
 
-const GazelleLanguageName = "aspect-configure"
+const GazelleLanguageName = "orion"
```

Comment:
This has never been documented but it will change what is outputted when running the cli `configure`, when it lists all the gazelle languages that are enabled.